php/Blocks/Controls/ControlAbstract.php

```diff
@@ -85,39 +85,34 @@ public function __construct() {
 	public function create_settings_config() {
 		$this->settings_config = [
 			'location'    => [
-				'name'     => 'location',
-				'label'    => __( 'Field Location', 'genesis-custom-blocks' ),
-				'type'     => 'location',
-				'default'  => 'editor',
-				'sanitize' => [ $this, 'sanitize_location' ],
```

Comment:
The `sanitize` key was from the PHP editor, where it sanitized the new values server-side.